infrastructure/irsa.tf

```diff
@@ -25,7 +7,7 @@ module "irsa_load_balancer" {
   oidc_providers = {
     one = {
       provider_arn               = module.iam_iam-github-oidc-provider.arn
-      namespace_service_accounts = ["vegait-load-balancer:load-balancer"]
+      namespace_service_accounts = ["${var.namespace_account}:load-balancer"]
```

Comment:
this should not be variable, put static namespace name and make sure you put the same in the lb helm chart when you deploy it

Reply:
aha, ok :)